statistics/cmsketch.go

```diff
@@ -145,7 +145,7 @@ func (c *CMSketch) calculateDefaultVal(helper *topNHelper, estimateNDV, scaleRat
 		c.defaultValue = 1
 	} else {
 		estimateRemainingCount := rowCount - (helper.sampleSize-uint64(helper.onlyOnceItems))*scaleRatio
-		c.defaultValue = estimateRemainingCount / (estimateNDV - uint64(sampleNDV) + helper.onlyOnceItems)
+		c.defaultValue = estimateRemainingCount / mathutil.MaxUint64(1, estimateNDV-uint64(sampleNDV)+helper.onlyOnceItems)
```

Comment:
If `estimateNDV` is equal to `sampleNDV`, the `defaultValue` should be `sampleSize / sampleNDV`?

Reply:
In this case, we assume that all the values are in the sample, so the default value should be 0 because it is for values that are no in the sample.